css/modern.css

```diff
@@ -690,6 +734,23 @@ img.splus-logo-inline {
     color: var(--text) !important;
 }
 
+#edge-filters-btn {
+    background-color: var(--text) !important;
+    color: var(--secondary) !important;
```

Comment:
Just making sure again that you meant to use a text color for the background and a background color for the text

Reply:
Yeah I did cause it's inverted